students/eric_rosko/lesson-01/iterator_2.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+'''
+Name:       iterator_2.py
+Author:     Eric Rosko
+Date:       Apr 8, 2018
+Python ver. 3.4.3
+'''
+
+from iterator_1 import IterateMe_1
+
+class IterateMe_2(IterateMe_1):
```

Comment:
not sure why you are inheriting `IterateMe_1` as you are overriding each function it has anyway.